src/main/scala/com/gu/support/workers/lambdas/SendThankYouEmail.scala

```diff
@@ -49,6 +49,7 @@ class SendThankYouEmail(thankYouEmailService: EmailService, servicesProvider: Se
           currency = c.currency,
           edition = state.user.country.alpha2,
           name = state.user.firstName,
+          product = s"${state.product.billingPeriod}-contribution",
```

Comment:
I would pass the billing period into the `ContributionEmailFields` class and generate this string there as there is less opportunity for a calling function to introduce variations that way. 

Also probably lowercase the billing period in the generated string to match with the previous hard coded casing.

Reply:
👍